dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -22,9 +24,10 @@ ENV DEBIAN_FRONTEND=noninteractive
 # TODO: remove libpython3.6-dev, its and pip's dependencies if we can get pip3 directly
 # install subagent
 # clean up
-RUN apt-get update && apt-get install -y libperl5.20 libpci3 libwrap0 \
-            libexpat1-dev \
-            curl gcc && \
+## TODO: remove click modules and grub2-common bash-completion if we can use PSU util directly
```

Comment:
Update TODO here, since you are now using the PSU plugin directly.

`TODO: remove grub2-common, bash-completion, python-pip, python-click-default-group, python-click, python-natsort and python-tabulate once we no longer need to install sonic-utilities for PSU plugin support`

Reply:
It's still required to install sonic-utilities for PsuBase accessing.

Reply:
Text changed